### PR TITLE
Add hystrixjs definition

### PR DESCRIPTION
**Pull request by igorsechyn**
Added a definition for the [hystrixjs](https://bitbucket.org/igor_sechyn/hystrixjs/overview) library

### CODE REVIEW

**Comment by vvakame** [comment]
*hystrixjs/hystrixjs.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/hystrixjs
  * http://bower.io/search/?q=hystrixjs
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (hystrixjs/hystrixjs-tests.ts or others)
* [ ] pass the Travis-CI test?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# DefinitelyTyped [![Build Status](https://travis-ci.org/borisyankov/DefinitelyTyped.png?branch=master)](https://travis-ci.org/borisyankov/DefinitelyTyped)
+# DefinitelyTyped [![Build Status](https://travis-ci.org/igorsechyn/DefinitelyTyped.png?branch=hystrixjs_definition)](https://travis-ci.org/igorsechyn/DefinitelyTyped)
```
Comment (vvakame) [inline README.md]: um? what's this?

**Comment by igorsechyn** [comment]
sorry about the readme travis ci badge. i have probably misunderstood the contribution guid about enabling the travis ci on my fork

**Comment by vvakame** [comment]
do not mind.
thanks mate!

**Pull request merged**